src/libsync/networkjobs.cpp

```diff
@@ -477,7 +477,8 @@ bool CheckServerJob::finished()
         if( status.contains("installed")
                 && status.contains("version")
                 && status.contains("versionstring") ) {
-
+            // Newer servers don't disclose anything in that empty string "version" anymore
```

Comment:
ah ok, so it still has the version string, but it is empty?
Shoud we not make it works also if the version string is not there at all?  (maybe future server remove it completely?)